# This document was autogenerated by the script camera_robot_calibration.py
# EDITING THIS FILE BY HAND IS NOT RECOMMENDED
# NOTE 1: This is the transformation between the camera coordinates and the robot reference system.
# NOTE 2: The pattern pose is the transformation of the calibration pattern wrt to the end-effector.
# More information: http://www.ensenso.com/manual/index.html?_cameras_byserialno_$stereoserial_link.htm

camera_left_robot:
  angle_axis: [2.6581026470626905, 0.6874331855628418, 0.6808250381967265, -0.25280997359944046]
  child: left/base_link
  duration: 230.546413899
  grid_spacing: 15.94
  iterations: 973
  parent: camera_optical_frame
  pattern_pose:
    rotation: [0.7161864681544047, 0.001856677420557697, -0.6978514634970954, 0.008765299563519747]
    translation: [0.024993163172439884, -0.0027451571511602985, 0.0060995670012445255]
  rotation: [0.6674438001357599, 0.6610278064326252, -0.24545869043735116, 0.23939725159656355]
  samples: 50
  seed:
    rotation: [0.6532814824381883, 0.6532814824381883, -0.27059805007309845, -0.2705980500730987]
    translation: [1.1750000000000003, 0.5289999999999999, 1.0000000000000004]
  translation: [-0.5246786665131972, -0.254436062874122, 1.514696999908204]
reprojection_error: {pixels: 1.68833683353835}
